SharedCryptoPkg/Driver/CommonSharedCryptoFull.c

```diff
@@ -0,0 +1,161 @@
+
+/** @file
+  This module is consumed by both DXE and SMM as well as PEI
```

Comment:
typo
Also lets put together a more descriptive description of this module.  What is "full".  What is in the c file

Reply:
Sorry, where's the typo?

Reply:
not seeing it anymore.  must have been a late night. :)